src/modules/decklink/decklink.cpp

```diff
@@ -60,14 +60,21 @@ std::vector<std::wstring> device_list()
         auto       pDecklinkIterator = create_iterator();
         IDeckLink* decklink;
         for (int n = 1; pDecklinkIterator->Next(&decklink) == S_OK; ++n) {
+#if defined(_MSC_VER)
+            BSTR m_name;
+#else
             String m_name;
+#endif        
             bool   success = SUCCEEDED(decklink->GetModelName(&m_name));
             decklink->Release();
             std::wstring model_name = L"Unknown";
 
-            if (success)
-                model_name = u16(m_name);
-
+            if (success && m_name != nullptr)
```

Comment:
Can this not use the get_model_name function defined in util.h?

Reply:
Yes we can